exampleCourse/serverFilesCourse/templates/collapsible_section.mustache.html

```diff
@@ -0,0 +1,18 @@
+<div class="card mb-1 mt-1">
+  <div class="card-header" style="cursor: pointer">
+    <h4
+      class="card-title d-flex justify-content-between"
+      data-toggle="collapse"
+      data-target="#{{section_name}}"
+    >
+      <div>{{section_header}}</div>
+      <div class="fa fa-angle-down"></div>
+    </h4>
```

Comment:
If we're going to stick with this example, I'd rather not use an `<h4>` here since this isn't underneath an `<h3>`. Plain text is fine.

Reply:
Switched this to a div (not 1000% sure this is correct syntax): https://github.com/PrairieLearn/PrairieLearn/pull/6747/commits/c34c7251e8702c9594174441044a5d108bf85466